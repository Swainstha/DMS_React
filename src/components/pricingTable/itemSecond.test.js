import testSnapFunction from '../../util/testSnapFunction';
import ItemSecond from './ItemSecond'
import React from 'react';
const dummy_data= {
    styleName:{width:2}, 
    headerStyle:{fontSize:3}, 
    itemStyle:{border:2}, 
    footerStyle:{color:"yellow"}
}

testSnapFunction("<itemSecond/>", "Snapshot test for PhroductGridItem",<ItemSecond {...dummy_data}/> )