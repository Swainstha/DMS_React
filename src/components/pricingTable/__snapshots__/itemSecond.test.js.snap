// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<itemSecond/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
